runtime/fiber.c

```diff
@@ -24,6 +24,20 @@
 #define fiber_debug_log(...)
 #endif
 
+void caml_change_max_stack_size (uintnat new_max_size)
```

Comment:
Is there a reason this was moved around in the file? That seems to complicate the diff which looks like it is only pulling Caml_state->current_stack in to a local variable?

Reply:
`caml_change_max_stack_size` has to move out of the `BYTE_CODE` ifdef block (which starts around line 225), that is why it has moved to the top of the file rather than duplicating it in the `NATIVE_CODE` and `BYTE_CODE` blocks.

Reply:
Ah, I missed that.